firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/UpdateTaskImpl.java

```diff
@@ -94,9 +97,9 @@ public UpdateTask addOnProgressListener(
       @Nullable Executor executor, @NonNull OnProgressListener listener) {
     ManagedListener managedListener = new ManagedListener(executor, listener);
     synchronized (lock) {
-      this.listener = managedListener;
+      this.listeners.add(managedListener);
```

Comment:
nit: remove `this.` here and in all other places where it's not explicitly needed

Reply:
Done!